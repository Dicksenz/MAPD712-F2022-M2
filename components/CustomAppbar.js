import React from "react";
import { View, StyleSheet, Text } from "react-native";

const CustomAppbar = ({ title }) => {
  return (
    <View style={styles.appbarContainer}>
      <Text style={styles.titleStyle}>{title}</Text>
    </View>
  );
};

const styles = StyleSheet.create({
  appbarContainer: {
    display: "flex",
    justifyContent: "flex-end",
    alignItems: "center",
    backgroundColor: "#347174",
    height: 100,
    borderBottomLeftRadius: 30,
    borderBottomRightRadius: 30,
  },

  titleStyle: {
    color: "white",
    fontSize: 20,
    marginBottom: 10,
  },
});

export default CustomAppbar;
