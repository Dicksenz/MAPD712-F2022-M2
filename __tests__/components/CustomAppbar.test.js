import React from "react";
import renderer from "react-test-renderer";
import CustomAppbar from "../../components/CustomAppbar";

test("Check if CustomAppbar component renders correctly", () => {
  const tree = renderer.create(<CustomAppbar />).toJSON();
  expect(tree).toMatchSnapshot();
});
