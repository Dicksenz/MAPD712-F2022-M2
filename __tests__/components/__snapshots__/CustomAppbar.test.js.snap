// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check if CustomAppbar component renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#347174",
      "borderBottomLeftRadius": 30,
      "borderBottomRightRadius": 30,
      "display": "flex",
      "height": 100,
      "justifyContent": "flex-end",
    }
  }
>
  <Text
    style={
      Object {
        "color": "white",
        "fontSize": 20,
        "marginBottom": 10,
      }
    }
  />
</View>
`;
